maestro-client/build.gradle

```diff
@@ -54,6 +54,7 @@ dependencies {
     implementation project(':maestro-ios')
     implementation 'com.google.code.findbugs:jsr305:3.0.2'
     implementation 'de.upb.cs.swt:axml:2.1.2'
+    implementation 'com.github.romankh3:image-comparison:4.4.0'
```

Comment:
I think we need to do api in this right? Folks implementing maestro-client otherwise will not get this artifact and we will face no class def found errors